src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -212,7 +229,13 @@ define([
 
         addColumnsForObject(telemetryObject) {
             let metadataValues = this.openmct.telemetry.getMetadata(telemetryObject).values();
+
+            this.addColunmName(telemetryObject, metadataValues);
             metadataValues.forEach(metadatum => {
+                if (metadatum.key === 'name') {
+                    return;
+                }
+
```

Comment:
Why are we overriding the original name from the metadata?

Reply:
We should use metadata name first, then domainObject name

Reply:
Yes It does that, `this.addColunmName(telemetryObject, metadataValues);` takes care of `Name` (as it need to be first column), so inside `metadataValues` loop ignoring `name`.

<img width="736" alt="Screen Shot 2020-10-23 at 9 36 12 AM" src="https://user-images.githubusercontent.com/1292612/97030125-37d1f280-1513-11eb-8505-c012dd98294a.png">